frontend/Peerprep/pages/index.vue

```diff
@@ -294,23 +287,16 @@ onUnmounted(() => {
                 Cancel Matching
               </Button>
             </div>
-            <!--
+
             <Button v-else class="w-3/4 mt-3"
               :disabled="isProcessing || matchFound || collaborationStore.isCollaborating">
               Match
             </Button>
-            -->
+            <Button type="button" @click="collaborationStore.clearCollaborationInfo">Clear</Button>
+
 
-            <Button v-else class="w-3/4 mt-3" :disabled="isProcessing">
-              Match
-            </Button>
```

Comment:
![image](https://github.com/user-attachments/assets/2153b97f-ee0d-458b-8ac5-ff487a7352f7)
Not sure what's the issue but UI has been broken

Reply:
fixed the position of the clear button